girder/__init__.py

```diff
@@ -35,6 +35,9 @@
 __license__ = 'Apache 2.0'
 
 
+Quiet = False
```

Comment:
If it's meant to cache a config value, let's make it internal for now: `_quiet`. We can easily open a symbol up for external use later if needed.